tests/services/test_service.py

```diff
@@ -104,3 +109,120 @@ def test_sorting_in_search(app, service, identity_simple, input_data):
 
     # Search with non existing sort parameter
     pytest.raises(ValidationError, service.search, identity_simple, sort="foo")
+
+
+def test_facets(app, service, identity_simple, input_data):
+    """Create a record."""
+    idx = 'records-record-v1.0.0'
```

Comment:
Minor: We smaller tests (in terms of lines of code). This will be hard to maintain long-term.